src/h5web/visualizations/heatmap/hooks.ts

```diff
@@ -67,3 +56,5 @@ export function useTextureData(
 
   return state;
 }
+
+export const useColorScaleDomain = createMemo(getColorScaleDomain);
```

Comment:
You did not want to keep the `Memo` in the hook name ? I found it handy to know that it was in fact a disguised `useMemo` hook

Reply:
Yeah, on second thought, you're right, it's clearer. It just felt weird at first :smile: